tests/global_setup.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Global setup for Jest test runner
+ *
+ * This is run once before any tests run and before the framework is
+ * installed into the runtime
+ */
+const knex = require('knex');
+const { getConfig } = require('../build/config');
+const { migrate } = require('../database');
+
+
+module.exports = async () => {
```

Comment:
No, but this is a _global_ setup. It runs once before any integration tests, and we know that at least one of those tests will touch the DB and the DB should be in a known state before that happens.

Reply:
My concern is that this will significantly slow down tests if we are truncating and reseeding the database when it’s not  needed. This may not be too noticeable now, but will get worse as our tests grow.